warehouse/models/mart/transit_database/_mart_transit_database.yml

```diff
@@ -508,6 +510,24 @@ models:
       - name: total_voms
       - name: volunteer_drivers
       - name: personal_vehicles
+  - name: dim_county_geography
+    description: '{{ doc("county_geography_table") }}'
+    columns:
+      - *key
```

Comment:
Curious if tests on this pass re: comment about unnesting organizations in the staging table -- I think there would be dups?